concepts/bids/bidder-deposit.mdx

```diff
@@ -24,7 +24,7 @@ Since deposits are split across $\text{blocksPerWindow} = 10$ L1 blocks, bidders
 
 ## Withdrawal Time
 
-The earliest withdrawal time can be calculated as the withdrawal start time, n-2 windows which is 30 L1 blocks (360 seconds) and oracle lag which is 20 L1 blocks (240 seconds) totalling 600 seconds, or 10 minutes. Therefore, the total time before funds can be withdrawn is approximately 50 blocks (10 minutes) from the initial deposit.
+The earliest withdrawal time can be calculated as the withdrawal start time, n-2 windows which is 30 L1 blocks (360 seconds) and oracle lag which is 10 L1 blocks (120 seconds) totalling 480 seconds, or 10 minutes. Therefore, the total time before funds can be withdrawn is approximately 40 blocks (8 minutes) from the initial deposit.
```

Comment:
The first sentence is confusing. Also, the 480 seconds should be 8 minutes instead of 10.

Reply:
Thank you, fixed!